index.html

```diff
@@ -6,8 +6,64 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <link rel="stylesheet" href="css/styles.css" />
+    <script
+      src="https://kit.fontawesome.com/da3dc3637a.js"
+      crossorigin="anonymous"
+    ></script>
   </head>
   <body>
-    <script src="https://kit.fontawesome.com/2ed0af19bb.js"></script>
+    <div class="wrapper">
+      <header>
+        <nav>
+          <ul>
+            <li><i class="fab fa-apple"></i></li>
+            <li>Store</li>
+            <li>Mac</li>
+            <li>iPod</li>
+            <li>iPhone</li>
+            <li>iPad</li>
+            <li>iTunes</li>
+            <li>Support</li>
+            <li>
+              <label>
+                <i class="fas fa-search"></i>
+                <input type="text" />
+              </label>
+            </li>
+          </ul>
+        </nav>
+        <div>
+          <a href="#">
+          <img id="verse" src="assets/verse.png" alt="verse" />
+          <img id="share" src="assets/share.png" alt="share" />
+          <img id="watch" src="assets/watch1.png" alt="watch" />
+          </a>
```

Comment:
- Use the `<main>` tag for the main content instead of the `<header>`. 
- Suggestion, you could use styled headings (especially an `<h1>` tag) instead of the `<img>` tags in lines 37 to 39, for a better SEO.